### DISCUSSION TITLE
Using actual fonts in SVG

### DISCUSSION THREAD

**Discussion by softhorizons** [metafloor/bwip-js#349] [Q&A]
Obviously rendering text to SVG as paths instead of <text> produces much fatter SVGs. That becomes an issue when emitting several thousand barcodes into a PDF as I'm doing.
Has anybody attempted to use <text> but found show-stoppers?

**Comment by softhorizons** [comment]
Sorry, looks like the text formatter played a trick on me. Should have been:
Obviously rendering text to SVG as paths instead of SVG `<text>` produces much fatter SVGs. That becomes an issue when emitting several thousand barcodes into a PDF as I'm doing.
Has anybody attempted to use `<text>` but found show-stoppers?

**Reply by metafloor** [reply]
With PDFs, you may be able to embed the font, and then reference through a `<text>` element.  Have never tried that but it's worth a shot if you are dealing with thousands of barcodes in a single document.

**Reply by softhorizons** [reply]
Thanks. I'll try it out, just wanted to make sure you hadn't already tried and run into a brick wall.
I'll keep you posted, will give you the code if it pans out.
